package.json

```diff
@@ -58,11 +59,13 @@
     "grunt-git-authors": "2.0.0",
     "grunt-html": "1.6.0",
     "grunt-jscs": "0.6.2",
+    "grunt-npm-copy": "0.1.0",
     "grunt-sass": "0.17.0",
     "grunt-selenium-server": "0.1.2",
     "grunt-svgmin": "2.0.0",
     "grunt-svgstore": "0.5.0",
     "load-grunt-tasks": "3.1.0",
+    "normalize.css": "3.0.2",
```

Comment:
You will already be able to update this one, Nicolas released v3.0.3 a few hours ago.